ProteGO/UI/History/SendHistoryProgress/ViewController/SendHistoryProgressViewController.swift

```diff
@@ -0,0 +1,45 @@
+import UIKit
+import RxSwift
+
+typealias SendHistoryProgressViewControllerBuilder = () -> SendHistoryProgressViewController
+
+protocol SendHistoryProgressViewControllerDelegate: class {
+    func sendHistoryFinished(result: Result<Void, Error>)
+}
+
+final class SendHistoryProgressViewController: UIViewController, CustomView {
```

Comment:
Może warto użyć bardziej generycznej nazwy i przenieść do `Common/Components`? Łatwiej byłoby użyć ten komponent w innych miejscach.

Reply:
Wydaje mi się ze to na tym etapie overengineering. Nie mamy żadnego designu, który by takiego dialogu wymagał, i wolałbym skupic się na innych aspektach appki na tym etapie.

Jak sie nam pojawi design z dialogiem podobym to wtedy bym wydzielił to do oddzielnego pliku